src/backend/gl/src/device.rs

```diff
@@ -424,6 +428,74 @@ impl Device {
             }
         }
     }
+
+    pub fn create_shader_program<'a>(
```

Comment:
are these changes a part of the windowing PR? or just refactoring accidentally got mixed in?

Reply:
I wanted to avoid re implementing create_shader_programs when making the shader for the swapchain quad stuff.